diff --git a/lib/gui.py b/lib/gui.py
index b717060..5e5e451 100644
--- a/lib/gui.py
+++ b/lib/gui.py
@@ -1930,7 +1930,7 @@ class gjots_gui:
             # locale version. Version could be 2.3.11.201012291355:
             v = gjots_version.split('.')
             vers = ".".join(v[0:3])
-            gjots_file_base = self.prefix + "/share/doc/gjots2-" + vers + "/gjots2"
+            gjots_file_base = self.prefix + "/share/doc/gjots2/gjots2"
             for env in "LANGUAGE", "LC_ALL", "LC_MESSAGES", "LANG":
                 locale_lang = ""
                 try:
diff --git a/setup.py b/setup.py
index 68b8e13..641de33 100755
--- a/setup.py
+++ b/setup.py
@@ -49,12 +49,12 @@ setup(
               'doc/gjots2.gjots','AUTHORS','README','INSTALL', 'COPYING', 'ChangeLog'
           ]
         ),
-	('share/doc/gjots2-' + gjots_version, ['doc/gjots2.en_US.gjots']),
-	('share/doc/gjots2-' + gjots_version, ['doc/gjots2.fr.gjots']),
-	('share/doc/gjots2-' + gjots_version, ['doc/gjots2.no.gjots']),
-	('share/doc/gjots2-' + gjots_version, ['doc/gjots2.nb.gjots']),
-	('share/doc/gjots2-' + gjots_version, ['doc/gjots2.ru.gjots']),
-	('share/doc/gjots2-' + gjots_version, ['doc/gjots2.es.gjots']),
+	('share/doc/gjots2-', ['doc/gjots2.en_US.gjots']),
+	('share/doc/gjots2-', ['doc/gjots2.fr.gjots']),
+	('share/doc/gjots2-', ['doc/gjots2.no.gjots']),
+	('share/doc/gjots2-', ['doc/gjots2.nb.gjots']),
+	('share/doc/gjots2-', ['doc/gjots2.ru.gjots']),
+	('share/doc/gjots2-', ['doc/gjots2.es.gjots']),
 	('share/pixmaps', ['pixmaps/gjots.png']),
 	('share/gjots2/ui', ['ui/gjots.ui']),
 	('share/gjots2/ui', ['ui/fileDialog.ui']),
index f170ee1..7343289 100644
--- a/doc/man/man1/gjots2.1
+++ b/doc/man/man1/gjots2.1
@@ -173,17 +173,7 @@ README, probably as something like:
 .RS
 .nf
 .ft CW
-gjots /usr/share/doc/gjots-0.8/gjots2.gjots
+gjots /usr/share/doc/gjots/gjots2.gjots
 .ft
 .RE
 .P
-or
-.P
-.RS
-.nf
-.ft CW
-netscape /usr/share/doc/gjots-0.8/gjots2.html
-.ft
-.fi
-.RE
-.P
