diff --git a/gjots2.desktop b/gjots2.desktop
index 1d41f97..b21066f 100644
--- a/gjots2.desktop
+++ b/gjots2.desktop
@@ -1,5 +1,4 @@
 [Desktop Entry]
-Encoding=UTF-8
 Name=Gjots2 Jotter
 Name[cs]=Zápisník Gjots2
 Name[en_US]=Gjots2 Jotter
@@ -19,5 +18,6 @@ Comment[ru]=Сохраняйте и организуйте Ваши заметк
 Exec=gjots2
 Terminal=false
 Type=Application
-Icon=gjots2.png
-Categories=GNOME;Application;Utility;
+Icon=gjots
+Categories=GNOME;GTK;Utility;
+Keywords=notes;hierarchy

